server/controllers/document.js

```diff
@@ -1,161 +1,147 @@
-(() => {
-  'use strict';
+import models from './../models/index';
```

Comment:
### Nitpick

You don't and shouldn't include the `/index` suffix when importing a module